content/blog/2016/2016-12-19-declarative-pipeline-beta.adoc

```diff
@@ -125,8 +125,8 @@ withEnv(["GIT_COMMITTER_NAME = jenkins","GIT_COMMITTER_EMAIL = jenkins@jenkins.i
 
 == How can you help?
 
-. Install the lastest version of the
- link:https://wiki.jenkins.io/display/JENKINS/Pipeline+Model+Definition+Plugin[_Pipeline-Model-Defintion_] plugin.
+. Install the latest version of the
+ link:/doc/book/pipeline/[_Pipeline-Model-Definition_] plugin.
```

Comment:
Specifically refers to the plugin, so https://plugins.jenkins.io/pipeline-model-definition/ would be better.

Reply:
totally changeable, but since the plugin linked to the book I figured it was easier to do directly to book.

Reply:
I mean, it doesn't really make a big difference either way at this point, but since it's about installing a new plugin, that would be more likely from the plugin site.

Reply:
Yea thats fair, I'll change it.